src/__tests__/restLink.ts

```diff
@@ -3630,7 +3630,7 @@ describe('export directive', () => {
       query userPostsWithTagDetails {
         user @rest(path: "/user") {
           id
-          posts {
```

Comment:
I'd prefer to see new behavior tested with a new test. It's been a while so I don't remember if this means we have less coverage of some other condition.

Would you mind instead copy-pasting this test and leaving the original test unchanged? (I'd prefer double-coverage of stuff over missing some case!